### PR TITLE
Adds two new Ballmer Peak messages

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Adds two new messages to the Ballmer Peak effect.

## Why It's Good For The Game
These messages reference important moments in SS13 history.

## Changelog
:cl:
add: Two new messages have been added to the Ballmer Peak status effect.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -430,7 +430,9 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 												"Best idea ever: Disposal pipes instead of hallways.",
 												"We should store bank records in a webscale datastore, like /dev/null.",
 												"You ever wonder if /dev/null supports sharding?",
-												"What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?"))
+												"What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?",
+												"We should totally invent DNA-locked disablers with a red alert emitter mode.",
```
Comment (BrumalLake) [inline code/modules/mob/living/carbon/life.dm]: wowww

totally also sounds sarcastic when it shouldn't be from their perspective and like a city girl or something

Comment (Archanial) [inline code/modules/mob/living/carbon/life.dm]: lol

File: code/modules/mob/living/carbon/life.dm [hunk 2]
```diff
@@ -430,7 +430,9 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
...
+												"Magical idea: Atmos pipes that only work 20% of the time!"))
```
Comment (BrumalLake) [inline code/modules/mob/living/carbon/life.dm]: words shouldn't be capitalized after colons
would also like it to reflect the optimization purpose of it more as well

Comment (ike709) [reply code/modules/mob/living/carbon/life.dm]: > words shouldn't be capitalized after colons

Incorrect. This is only the case for introducing lists and in bad style guides.

File: code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (ike709) [inline code/modules/mob/living/carbon/life.dm]: Decent idea, terrible wording.

Comment (Ivniinvi) [reply code/modules/mob/living/carbon/life.dm]: Reworded

File: code/modules/mob/living/carbon/life.dm (hunk 2 above)
Comment (Ivniinvi) [reply code/modules/mob/living/carbon/life.dm]: Reworded

Comment (BrumalLake) [reply code/modules/mob/living/carbon/life.dm]: but atmos pipes that only work 20% of the time is a dependent clause

**Pull request closed without merging**